workspace/viewer/PdfJsViewer.ts

```diff
@@ -515,49 +520,51 @@ export class PdfJsViewer {
 
   onResize() {
     if (this.currentFile) {
-      this.openCurrentPage();
+      this.showPage(this.currentPage);
     }
   }
 
   pageUp = () => {
-    if (this.currentPage > 1) {
-      this.currentPage -= 1;
-      this.openCurrentPage();
-    }
+      if (this.currentPage > 1) {
+          this.currentPage -= 1;
+          this.showPage(this.currentPage);
+      }
   };
 
   pageDown = () => {
-    if (this.currentFile && this.currentPage < this.currentFile.numPages) {
-      this.currentPage += 1;
-      this.openCurrentPage();
-    }
+      if (this.currentFile && this.currentPage < this.currentFile.numPages) {
+          this.currentPage += 1;
+          this.showPage(this.currentPage);
+      }
   };
 
-  getCurrentPage(): number | undefined { return this.currentPage; }
+  getCurrentPage(): number | undefined {
+    return this.currentPage;
+  }
 
   getZoomScale(): number { return this.zoom.current(); }
 
   // Toolbar button handlers
   zoomIn = () => {
-    this.zoom.zoomIn() && this.openCurrentPage();
+    this.zoom.zoomIn() && this.showPage(this.currentPage);
   };
 
   zoomOut = () => {
-    this.zoom.zoomOut() && this.openCurrentPage();
+    this.zoom.zoomOut() && this.showPage(this.currentPage);
   };
 
   zoomWidth = () => {
     this.zoom.setFitting(ZoomingMode.FIT_WIDTH);
-    this.openCurrentPage();
+      this.showPage(this.currentPage);
```

Comment:
Something is wrong with the indents here.

Reply:
Done.